src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -0,0 +1,46 @@
+<%@ page import="java.util.List" %>
+<%@ page import="codeu.model.data.Conversation" %>
+<%@ page import="codeu.model.data.Message" %>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+
+<html>
+<head>
+  <title>CodeU Chat App</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
```

Comment:
I'm also wondering where do you think the profile page should be accessible from? I'm doing admin page and was trying to figure out if it should be a tab on the top or the first thing that comes up when you log in? Currently it's just accessible by link is that correct?